### DISCUSSION TITLE
Update to Netbox 4.0.8 failed

### DISCUSSION THREAD

**Discussion by fbaschin** [netbox-community/netbox#17085] [Help Wanted!]
### Deployment Type

NetBox Cloud

### NetBox Version

4.0.8

### Python Version

3.10

### Steps to Reproduce

I try to Update Netbox to version 4.0.8 from 3.7.4 von Debian 11

I installed pythons 3.10 before und start
PYTHON=/usr/local/bin/python3.10 ./upgrade.sh

And get this error

RuntimeError: Model class netbox_dns.models.record.Record doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.

### Expected Behavior

correct install

### Observed Behavior

error message

**Comment by candlerb** [comment]
You appear to have installed a third-party plugin called `netbox_dns`. Remove it from PLUGINS in configuration.py then try again.

If you want to re-enable the functionality, talk to the plugin author about making it compatible with Netbox 4.0

**Reply by fbaschin** [reply]
after disable DNS Plugin it now works
